types/react-column-resizer/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for react-column-resizer 1.1
+// Project: https://github.com/nik-m2/react-column-resizer#readme
+// Definitions by: ibrahim <https://github.com/ibrahim-13>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { Component } from 'react';
+
+interface ResizerProps {
```

Comment:
```suggestion
iexport nterface ResizerProps {
```
it will make `export {}` redundant